py/runai_model_streamer/runai_model_streamer/file_streamer/file_streamer.py

```diff
@@ -1,5 +1,6 @@
 import os
 from typing import List, Iterator
+import numpy as np
```

Comment:
Can we import just the empty. i.e. `from numpy import empty`

Reply:
In python, importing partial package does not reduce import time (https://stackoverflow.com/questions/710551/use-import-module-or-from-module-import).

Its mainly convention.

I prefer keeping `import numpy as np` because it keep us with the context.
I mean comparing `np.empty(...` vs `empty(...`
In my opinion its more convenient to see the first as you immediately understand its numpy array.
In addition to that, it became a convention in the community to import numpy as np

For example, can you imagine people are doing `from torch import zeros` and using `zeros(...` instead of `torch.zeros(...` in the middle of the code?